Utils/AppConstants.swift

```diff
@@ -4,14 +4,14 @@
 
 public struct AppConstants {
     static var StatusBarHeight: CGFloat {
-        if UIApplication.sharedApplication().statusBarHidden {
+        if UIScreen.mainScreen().traitCollection.verticalSizeClass == .Compact {
             return 0
         }
         return 20
     }
-    static let ToolbarHeight: CGFloat = 44
-    static let DefaultRowHeight: CGFloat = 58
 
+    static var ToolbarHeight: CGFloat = 44
```

Comment:
Heh. This number should be dynamic (and smaller in landscape). I changed it here once, but changed it back when things got hairy. Forgot the let->var change.